examples/lzapp-migration/layerzero.config.ts

```diff
@@ -1,88 +1,126 @@
 import { EndpointId } from '@layerzerolabs/lz-definitions'
+import { ExecutorOptionType } from '@layerzerolabs/lz-v2-utilities'
 
 import type { OAppOmniGraphHardhat, OmniPointHardhat } from '@layerzerolabs/toolbox-hardhat'
 
 const sepoliaContract: OmniPointHardhat = {
-    eid: EndpointId.SEPOLIA_TESTNET,
-    contractName: 'MyLzApp',
+    eid: EndpointId.SEPOLIA_TESTNET, /// EndpointV1
+    contractName: 'MyEndpointV1OFTV2Mock', // Update this if trying out the other contracts
 }
 
-const arbSepoliaContract: OmniPointHardhat = {
-    eid: EndpointId.ARBSEP_V2_TESTNET,
-    contractName: 'MyOApp',
+const solanaContract: OmniPointHardhat = {
+    eid: EndpointId.SOLANA_V2_TESTNET,
+    address: '', // NOTE: update this with the OFTStore address.
 }
 
+// The values here are for development purposes. E.g. confirmations are set to 1. For production, they should be reviewed and edited accordingly.
 const config: OAppOmniGraphHardhat = {
     contracts: [
         {
             contract: sepoliaContract,
         },
         {
-            contract: arbSepoliaContract,
+            contract: solanaContract,
         },
     ],
     connections: [
         {
-            from: arbSepoliaContract,
-            to: sepoliaContract,
+            from: sepoliaContract,
+            to: solanaContract,
             config: {
-                sendLibrary: '0x4f7cd4DA19ABB31b0eC98b9066B9e857B1bf9C0E',
+                sendLibrary: '0x6862b19f6e42a810946B9C782E6ebE26Ad266C84', // SendULN301 on Ethereum Sepolia
                 receiveLibraryConfig: {
-                    receiveLibrary: '0x75Db67CDab2824970131D5aa9CECfC9F69c69636',
+                    receiveLibrary: '0x5937A5fe272fbA38699A1b75B3439389EEFDb399', // ReceiveULN301 on Ethereum Sepolia
                     gracePeriod: BigInt(0),
                 },
                 sendConfig: {
                     executorConfig: {
                         maxMessageSize: 10000,
```

Comment:
maxMessageSize is still likely less than 10K to solana.

Reply:
Will remove this line.